index.js

```diff
@@ -145,44 +145,42 @@ else if (program.hub) {
         helpers.logError(error);
     });
 }
-else if(program.refreshAuthToken){
+else if (program.refreshAuthToken) {
     console.log("------ Refreshing oAuth token for hub %j".header, program.refreshAuthToken);
     var onboardingCli = new OnboardingCli();
-    onboardingCli.loadTranslatorAndGetOnboardingAnswers(program.refreshAuthToken).then(answers => {
-        var fileName = helpers.createOnboardingFileName(program.refreshAuthToken);
-        helpers.readFile(fileName, "Please complete onboarding -o").then(data => { 
+    var fileName = helpers.createOnboardingFileName(program.refreshAuthToken);
+    helpers.readFile(fileName, "Please complete onboarding -o").then(data => {
         let configInfo = JSON.parse(data);
         let authInfo = configInfo.authInfo;
-        translatorCli.getProperty(program.refreshAuthToken, authInfo, 'refreshAuthToken', answers).then(refreshedInfo => {
-            helpers.logObject(refreshedInfo);
-            console.log("------ Saving refreshed onboardingInfo to: " + fileName); 
-            let configData = helpers.createConfigData(configInfo.translator, configInfo.translatorPackageName, refreshedInfo);
-            let refreshedData = JSON.stringify(configData);
-            fs.writeFile(fileName, refreshedData, function (err) {
-                if (err) {
-                    console.log(err);
-                    return console.log(err);
-                }
-            console.log("Saved!");
-        });
+        onboardingCli.loadTranslatorAndGetOnboardingAnswers(configInfo.translatorPackageName).then(answers => {
+            translatorCli.getProperty(configInfo.translatorPackageName, authInfo, 'refreshAuthToken', answers).then(refreshedInfo => {
+                helpers.logObject(refreshedInfo);
+                console.log("------ Saving refreshed onboardingInfo to: " + fileName);
+                let configData = helpers.createConfigData(configInfo.translator, configInfo.translatorPackageName, refreshedInfo);
+                let refreshedData = JSON.stringify(configData);
+                fs.writeFile(fileName, refreshedData, function (err) {
+                    if (err) {
+                        console.log(err);
+                        return console.log(err);
```

Comment:
so you are logging and then returning what console.log() returns ? I am not sure what is happening here